src/signals/incident-management/containers/DashboardContainer/saga.js

```diff
@@ -0,0 +1,45 @@
+import { all, put, takeLatest } from 'redux-saga/effects';
+import { pullAllBy } from 'lodash';
+import { authCall } from 'shared/services/api/api';
+import CONFIGURATION from 'shared/services/configuration/configuration';
+
+import {
+  REQUEST_DASHBOARD,
+  UPDATE_DASHBOARD
+} from './constants';
+import {
+  requestDashboardSuccess,
+  requestDashboardError
+} from './actions';
+
+export const requestURL = `${CONFIGURATION.API_ROOT}signals/experimental/dashboards/1`;
+
+export function* fetchDashboard() {
+  try {
+    const dashboard = yield authCall(requestURL);
+
+    dashboard.hour = dashboard.hour.map((item) => ({
+      ...item,
+      timestamp: new Date(item.interval_start).getTime()
+    }));
+
+    // TEMP: remove these items
```

Comment:
Should this be removed later or now?

Reply:
we do not get the timestamp from the server. only date string. what do you suggest?